Daemon-CertificateCredential-DotNet.sln

```diff
@@ -1,15 +1,18 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.26228.0
+VisualStudioVersion = 15.0.26228.4
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TodoListService", "TodoListService\TodoListService.csproj", "{047CDEF7-D5BA-4B1E-9748-910B610CCA51}"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TodoListDaemonWithCert", "TodoListDaemonWithCert\TodoListDaemonWithCert.csproj", "{ACDF371E-8F3C-4F2B-8F29-AB22074D91FD}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{C7F44F54-B18B-4830-8A19-6FA17378D7B0}"
 	ProjectSection(SolutionItems) = preProject
```

Comment:
Added the new scripts and the README.md file in the solution items so that users of the sample see them and inspect them if they wish.